app/assets/stylesheets/ui_components/select.scss

```diff
@@ -0,0 +1,8 @@
+.form-inline [data-react-class="ui_components.Select"] {
+  display: inline-block;
+  vertical-align: middle;
+
+  .search-field > input {
+    min-width: 222px;
```

Comment:
The bigger the field, the harder to miss it. So, how about time between the DOM ready event and first keystroke as a KPI?